inst/validation/utils-validation.R

```diff
@@ -0,0 +1,148 @@
+#' Setting up the validation
+#'
+#' 1. Add package_name
+#' 2. Copy that variable and the contents of if block to tests/testthat/setup.R
+#' (If you are using the template this may already be in place for you)
+
+package_name <- "dv.templates"
```

Comment:
could we use read.dcf("DESCRIPTION")[1, "Package"] here?

Reply:
For the moment the qc documentation works like this by using the hardcoded name. This documentation is not consumed in the package calling the workflows in dv.template those will use their own copy in their own repo. Agreed that we can consider including and consuming the qc documentation from dv.templates in the future but this PR does not cover that now.